pkg/clustermanager/reconcile_logicalcluster.go

```diff
@@ -88,15 +88,14 @@ func (c *controller) processAddLC(newCluster *lcv1alpha1.LogicalCluster) error {
 	// Create cluster
 	var opts pclient.Options
 	//ES: what exactly is this kubeconfig
-	createdCluster, err := provider.Create(c.context, clusterName, opts)
+	err = provider.Create(c.context, clusterName, opts)
 	if err != nil {
 		logger.Error(err, "failed to create cluster")
 		return err
 	}
 	logger.Info("Done creating cluster", "clusterName", clusterName)
 
 	// Update the new cluster's status - specifically the config string and the phase
```

Comment:
Please update the comment. We are not updating the config string.

Reply:
done